src/serlo-editor/plugins/rows/editor.tsx

```diff
@@ -22,6 +18,9 @@ export function RowsEditor({ state, config, id, editable }: RowsProps) {
     })
   }
 
+  const showLargeAddButton =
+    parentType === EditorPluginType.Article || parentType?.startsWith('type-')
```

Comment:
`parentType?.startsWith('type-')` deserves a comment imo, most people (including me) probably don't understand the logic behind this.

Reply:
I hope you don't actually feel like you have to plea for documentation :D
put it into a variable to explain what it does. Okay like this?

Reply:
I think I just misused "plea", I thought it means "ask nicely", my bad 😄 

Looks great to me!